docker/psi4/src/describe.py

```diff
@@ -0,0 +1,43 @@
+def get_description():
+    return {
+        "name": "PSI4",
```

Comment:
This looks like we could just put it in a JSON file, edit it there, and let the Python code load it from the image and return it. Move it to describe.json perhaps?